src/Sitemap/SitemapFactory.php

```diff
@@ -220,23 +215,6 @@ private function createImagesSitemap(): string
         return $images->get();
     }
 
-    private function createVideosSitemap(): string
-    {
-        $images = $this->cache->getItem('sitemap_videos');
-
-        if (!$images->isHit()) {
-            $sitemap = new Sitemap();
-            $this->addVideos($sitemap);
-
-            $images->set((string) $sitemap);
-            $images->expiresAfter(self::EXPIRATION_TIME);
-
-            $this->cache->save($images);
-        }
-
-        return $images->get();
-    }
-
     private function addArticlesCategories(Sitemap $sitemap)
```

Comment:
missing return type (return types are missing for all these addXXX() methods) (not related to your PR, i know \o/)